src/devices/feibit.ts

```diff
@@ -143,6 +143,123 @@ const definitions: Definition[] = [
+        vendor: 'Feibit',
+        description: 'Curtain motor',
+        extend: [windowCovering({controls: ['lift', 'tilt']})],
+    },
+    {
+        zigbeeModel: ['FB56+TMT01ZY1.6'],
+        model: 'SSS401ZB-T',
+        vendor: 'Feibit',
+        description: 'Temperature control panel',
+        fromZigbee: [fz.thermostat, fz.on_off, fz.fan],
+        toZigbee: [
+            tz.thermostat_local_temperature,
+            tz.thermostat_occupied_heating_setpoint,
+            tz.thermostat_occupied_cooling_setpoint,
+            tz.thermostat_system_mode,
+            tz.on_off,
+            tz.fan_mode,
+        ],
+        exposes: [
+            e
+                .climate()
+                .withLocalTemperature()
+                .withSystemMode(['off', 'auto', 'cool', 'heat', 'fan_only', 'dry'])
+                .withFanMode(['off', 'low', 'medium', 'high', 'on', 'auto'])
+                .withSetpoint('occupied_heating_setpoint', 5, 30, 0.5)
+                .withSetpoint('occupied_cooling_setpoint', 5, 30, 0.5),
+            e.switch(),
+        ],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['hvacThermostat', 'hvacFanCtrl', 'genOnOff']);
+            await reporting.thermostatOccupiedHeatingSetpoint(endpoint);
+            await reporting.thermostatTemperature(endpoint);
+            await reporting.fanMode(endpoint);
+            await reporting.onOff(endpoint);
+        },
+    },
+    {
+        zigbeeModel: ['FB56+ZSC02KJ1.0'],
+        model: 'TZSC302W-V1',
+        vendor: 'Feibit',
+        description: 'Dimmer Switch',
+        extend: [deviceEndpoints({endpoints: {'11': 11, '12': 12, '13': 13}}), light({powerOnBehavior: false})],
+    },
+    {
+        zigbeeModel: ['FTB56+SKT22HY1.1'],
+        model: 'TSKT222W-H4',
+        vendor: 'Feibit',
+        description: 'Power socket with metering',
+        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering],
+        toZigbee: [tz.on_off, tz.power_on_behavior],
+        exposes: [e.switch(), e.power(), e.energy()],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'haElectricalMeasurement', 'seMetering']);
+            await reporting.onOff(endpoint);
+            await reporting.readEletricalMeasurementMultiplierDivisors(endpoint);
+            await reporting.activePower(endpoint);
+            await reporting.readMeteringMultiplierDivisor(endpoint);
+            await reporting.currentSummDelivered(endpoint);
+        },
+    },
+    {
+        zigbeeModel: ['FB56+CUR18SB2.0'],
+        model: 'TCUR218W-V1',
+        vendor: 'Feibit',
+        description: 'Dual track curtain controller',
+        extend: [deviceEndpoints({endpoints: {bottom: 14, top: 15}}), identify()],
```

Comment:
Please try to use `windowCovering` instead of `fromZigbee`/`toZigbee`/`exposes`

Reply:
OK，But I have two sets of controls. How should I configure them?

Reply:
![image](https://github.com/user-attachments/assets/6ba15190-d9cc-4185-b25f-6dde92987efb)

Reply:
Added support for this, you can use `endpointNames` now (https://github.com/Koenkk/zigbee-herdsman-converters/commit/5a3af85dd57a008fab09134207d403d5745bd34c)

Reply:
OK, Will be like
```js
  zigbeeModel: ['FB56+CUR18SB2.0'],
  model: 'TCUR218W-V1',
  vendor: 'Feibit',
  description: 'Dual track curtain controller',
  extend: [
      deviceEndpoints({endpoints: {top: 15, bottom: 14}}),
      identify(),
      windowCovering({controls: ['lift', 'tilt'], endpointNames: ['top', 'bottom']}),
  ],
```

Thx

Reply:
Fixed in https://github.com/Koenkk/zigbee-herdsman-converters/commit/8455cf3945b20dc9864a732a3a8f7d7dccf7ac63

Reply:
Think you~